html/manifest/file_handlers.json

```
@@ -0,0 +1,44 @@
+{
+  "html": {
+    "manifest": {
+      "file_handlers": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/file_handlers",
+          "spec_url": "https://wicg.github.io/manifest-incubations/index.html#file_handlers-member",
+          "support": {
+            "chrome": {
+              "version_added": "102"
+            },
+            "chrome_android": {
+               "version_added": false
+             },
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": {
+              "version_added": false
+            }
```

Apparently CI wants us to use "mirror" for webview_android given it matches chrome_android now.

```suggestion
            "webview_android": "mirror",
```